### PR TITLE
Convert buttons to a list of links

### LINKED ISSUE #25: Mysterious Line on Mouse Hover
Mouse hover produces black line in some cases

screen shot 2018-04-10 at 3 31 02 pm

Happens for every button in this group except for the last one

screen shot 2018-04-10 at 3 32 36 pm

### PR DESCRIPTION
**Pull request by mboneil10**
Closes #25 

This is what it looks like now
screen shot 2018-05-22 at 10 47 40 am

### CODE REVIEW

File: app/views/bus_stops/by_route.haml [hunk 1]
```diff
@@ -3,6 +3,8 @@
 - if @stops.empty?
   No stops on this route
 - @stops.each do |stop|
-  = link_to edit_bus_stop_path(stop.hastus_id) do
-    %button.btn.btn-default= stop.name
+  %ul
```
Comment (dfaulken) [inline app/views/bus_stops/by_route.haml]: A ul per stop isn't right. Should be one ul and a li per stop.

Comment (mboneil10) [reply app/views/bus_stops/by_route.haml]: oopsies

File: app/views/bus_stops/by_route.haml [hunk 2]
```diff
@@ -2,7 +2,9 @@
 
 - if @stops.empty?
   No stops on this route
+  %ul
 - @stops.each do |stop|
-  = link_to edit_bus_stop_path(stop.hastus_id) do
-    %button.btn.btn-default= stop.name
+  %li
+    = link_to edit_bus_stop_path(stop.hastus_id) do
+      - stop.name
```
Comment (dfaulken) [inline app/views/bus_stops/by_route.haml]: Indentation still isn't right.

```haml
%ul
  - @stops.each
    %li
```

Comment (mboneil10) [reply app/views/bus_stops/by_route.haml]: goddammit thanks Dave

**Comment by mboneil10** [comment]
Everything's fine.

**Pull request merged**